googlemaps/client.py

```diff
@@ -272,6 +287,10 @@ def _generate_auth_url(self, path, params, accepts_clientid):
 from googlemaps.roads import snap_to_roads
 from googlemaps.roads import speed_limits
 from googlemaps.roads import snapped_speed_limits
+from googlemaps.places import places
+from googlemaps.places import place
+from googlemaps.places import photo
+from googlemaps.places import autocomplete
```

Comment:
All of the other modules basically just have one method, though. Places is a much bigger API.